Mathlib/Analysis/SpecialFunctions/Trigonometric/Arctan.lean

```diff
@@ -200,6 +200,83 @@ theorem arccos_eq_arctan {x : ℝ} (h : 0 < x) : arccos x = arctan (sqrt (1 - x
   · linarith only [arcsin_pos.2 h]
 #align real.arccos_eq_arctan Real.arccos_eq_arctan
 
+theorem arctan_inv_of_pos {x : ℝ} (h : 0 < x) : arctan x⁻¹ = π / 2 - arctan x := by
```

Comment:
Can to put these lemmas into a separate section, as appropriate, and mention them in the module docstring?

Reply:
Done.